internal/store/pod.go

```diff
@@ -587,19 +587,19 @@ func createPodInitContainerInfoFamilyGenerator() generator.FamilyGenerator {
 		metric.Gauge,
 		"",
 		wrapPodFunc(func(p *v1.Pod) *metric.Family {
-			ms := make([]*metric.Metric, len(p.Status.InitContainerStatuses))
+			var ms []*metric.Metric
```

Comment:
Nit for consistency with #1731, but really optional:
```suggestion
			ms := []*metric.Metric{}
```